generate/generate-metadata.sh

```diff
@@ -49,16 +49,12 @@ for src in dev compliance-tool; do
   fi
   
   # sign
-  if test -z `which xmlsectool`; then
-    echo "$(tput setaf 3)Installing xmlsectool$(tput sgr0)"
-    brew install xmlsectool
-  fi
-  
   echo "$(tput setaf 3)Signing metadata$(tput sgr0)"
-  xmlsectool \
+  $XMLSECTOOL \
```

Comment:
```suggestion
  xmlsectool \
```
I think you can leave this as is. The value gets conditionally set on line 9, so should pick up the ENV variable if set, or default to just `xmlsectool`

Reply:
Set as a value... Which in bash has to be accessed as a variable ($VAR)